File: Dia54/ValidadorURLs.py
# Validador de URLs en Python

import re

REGEX = r'^https?:\/\/(\w+\.)+(\w+)(\/\w+)*(\/|\?(\w+=(\w|-)+&)*(\w+=(\w|-)+)|#.+)?$'

def isUrl(url):
  return re.match(REGEX, url) != None


print(isUrl('https://www.platzi.com')) # true
print(isUrl('hello friends')) # false

File: Dia54/ValidadorURLs.js
// Validador de URLs en JavaScript

const REGEX = /^https?:\/\/(\w+\.)+(\w+)(\/\w+)*(\/|\?(\w+=(\w|-)+&)*(\w+=(\w|-)+)|#.+)?$/g

function isUrl(url) {
  return REGEX.test(url)
}

console.log(isUrl('https://www.platzi.com')) // true
console.log(isUrl('hello friends')) // false